src/gwt/panmirror/src/editor/src/optional/ace/ace.ts

```diff
@@ -367,6 +252,9 @@ class CodeBlockNodeView implements NodeView {
   }
 
   public setSelection(anchor: number, head: number) {
+    if (!this.aceEditor || !this.editSession) {
```

Comment:
When the underlying Ace editor is actually created, do we need to forward the selection in the case that the user has managed to logically get the selection there before it has been rendered?